code/processes/filealerter.q

```diff
@@ -41,7 +42,24 @@ find:{[path;match]
 	.lg.o[`alerter;"searching for ",path,"/",match];
 	files:@[system;findstring;()];
 	if[os=`win;files:,/:[path,"\\"; files]]; files};
+
+//-decodes pcap file and sends to tickerplant
+processpcaps:{[path;file]
+	.lg.o[`alerter;"processing pcap file"];
+	.lg.o[`alerter;"decoding pcap file"];
+	table: .pcap.buildtable[hsym `$(path,"/",file)];
+	
+	.lg.o[`alerter;"checking connection to tickerplant"];
+	if[not tickerplanttype in .servers.SERVERS[`proctype]; .servers.startup[]];
+	if[not tickerplanttype in .servers.SERVERS[`proctype]; .lg.e[`alerter;"connection to tickerplant failed"]; :()];
 	
+	.lg.o[`alerter;"sending table to tickerplant"];
+	sendtotickerplant[tickerplanttype;`packets;table[cols table]]
```

Comment:
I think the name of the table should be a parameter to the function (and that would also then influence the FSP changes)

Reply:
is it possible to add a third parameter to file alerter functions? the documentation seems to suggest that these functions should only take file and path since you dont specify actual parameter values in the csv

Reply:
I think the way to do it is have this function take a third argument, and then in FSP create a project in code/filealerter which creates a new function with the table name projected